### DISCUSSION TITLE
两种tinymce-plugin安装命令都执行了，但是在页面调用的时候还是报错了，

### DISCUSSION THREAD

**Discussion by zhxde** [tinymce-plugin/tinymce-plugin.github.io#16] [Q&A(问题求助)]
![image](https://user-images.githubusercontent.com/56284349/199911398-ff3b70cf-2589-4c9d-9984-07772b0c1f7a.png)

**Comment by Five-great** [comment]
@zhxde  你可以见 @ #12
